### PR TITLE
MHP-2167 survey form success message

### PR DESCRIPTION
**Pull request by mikeallisonJS**

### CODE REVIEW

File: app/assets/javascripts/angular/components/publicSurvey/publicSurvey.html [hunk 1]
```diff
@@ -25,7 +25,8 @@
                 <div class="pa4 tl">
                     <h1 class="pivot-gray-darker f2 mt3 normal mb0">{{ $ctrl.survey.title }}</h1>
                     <p class="lh-copy mt1" ng-if="!$ctrl.thankYou">{{ $ctrl.survey.login_paragraph }}</p>
-                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou">{{ $ctrl.survey.post_survey_message }}</p>
+                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou && $ctrl.survey.post_survey_message">{{ $ctrl.survey.post_survey_message }}</p>
+                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou && !$ctrl.survey.post_survey_message">{{ 'surveys:publicView.post_survey_message' | t }}</p>
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/publicSurvey/publicSurvey.html]: Would just having one of these be cleaner? Hopefully my Angular template syntax works... Maybe I'm expecting too much from Angular...
```suggestion
                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou">{{ $ctrl.survey.post_survey_message || ('surveys:publicView.post_survey_message' | t) }}</p>
```

Comment (mikeallisonJS) [reply app/assets/javascripts/angular/components/publicSurvey/publicSurvey.html]: It 'should' have worked. It seems the il8next library is finicky, ran into this weirdness earlier

Comment (OzzieOrca) [reply app/assets/javascripts/angular/components/publicSurvey/publicSurvey.html]: Huh, weird. Whatever. Does it throw errors or just not render anything?

Comment (mikeallisonJS) [reply app/assets/javascripts/angular/components/publicSurvey/publicSurvey.html]: renders nothing

**Pull request merged**